src/gas/gas-util.test.ts

```diff
@@ -50,7 +56,75 @@ const mockEIP1559ApiResponses = [
   },
 ];
 
-describe('gas utils', () => {
+describe('gas-util', () => {
+  describe('normalizeGWEIDecimalNumbers', () => {
```

Comment:
The tests for `normalizeGWEIDecimalNumbers` were already present but were moved up so that when comparing this file to `gas-util.ts` side-by-side, it's easier to verify that there is test coverage for this function, as the `describe`s in the test file are now listed in the same order as the functions are listed in `gas-util.ts`.